src/js/edu-benefits/definitions/postHighSchoolTrainings.js

```diff
@@ -44,7 +44,7 @@ const uiSchema = {
       'ui:title': 'Degree, diploma or certificate received'
     },
     major: {
-      'ui:title': 'Major or course of study (NOT for high school)'
+      'ui:title': 'Major or course of study'
```

Comment:
This will conflict with https://github.com/department-of-veterans-affairs/vets-website/pull/5118, but that's fine.